src/components/Calendar/EditSchedule.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Tooltip, Button, Menu } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { Edit } from '@material-ui/icons';
+import ScheduleNameDialog from './ScheduleNameDialog';
+import DeleteScheduleDialog from './DeleteScheduleDialog';
+
+const styles = () => ({
+    editButton: {
+        padding: '3px 7px',
+        minWidth: 0,
+        minHeight: 0,
+    },
+});
+
+const EditSchedule = (props) => {
+    const [anchorEl, setAnchorEl] = useState(null);
```

Comment:
Deference classes from props up here so that later on, you can just do `classes.<style>` instead of `props.classes.<style> over and over again.
```suggestion
const EditSchedule = (props) => {
    const { classes } = props;

    const [anchorEl, setAnchorEl] = useState(null);
```